PHPCI/Plugin/Util/TestResultParsers/Codeception.php

```diff
@@ -50,10 +50,10 @@ public function parse()
 
             foreach ($testsuite->testcase as $testcase) {
                 $testresult = array(
-                    'suite' => (string) $testsuite['name'],
+                    'suite' => ucfirst((string) $testsuite['name']),
                     'file' => str_replace($this->phpci->buildPath, '/', (string) $testcase['file']),
                     'name' => (string) $testcase['name'],
-                    'feature' => (string) $testcase['feature'],
+                    'feature' => ucfirst((string) $testcase['feature']),
```

Comment:
Just for pretty output. "feature" attribute is taken by Codeception from the method name and in most cases this method name is started from lower case letter.